pkg/broker/server/sse.go

```diff
@@ -167,7 +181,7 @@ func (b *Broker) run() {
 			// Disconnect all clients
 			b.clients.Walk(
 				func(key string, value interface{}) bool {
-					chset, ok := value.(map[chan []byte]int)
+					chset, ok := value.(map[chan []byte]clientValue)
```

Comment:
I wrote this because I was not sure how to use the original `int`.  Fixed, thanks!